### PR TITLE
TD - 3037 Update status card

### PR DESCRIPTION
**Pull request by m4manjesh**
Closes [TD-3037](https://sce.myjetbrains.com/youtrack/issue/TD-3037/Update-StatusCard)

## Changes

- `Operational` and `Disrupted` statuses added to `statuses.ts`
- `StatusCard`  title and status `typography` updated as per Figma
- `Tooltip` added to `StatusIcon` with optional `title` prop
- New stories added for `tooltip`
- `StatusIcon` tests updated for `tooltip`

## UI/UX
Figma: https://www.figma.com/design/5qkWskMGpXY38REVIDcJnW/Landing-Page?node-id=3559-32542&node-type=FRAME&t=usvKg62evL72zXSb-0

New statuses
Screenshot 2024-09-06 161002
Screenshot 2024-09-06 160949

Status Icon updated style with tooltip
Screenshot 2024-09-06 160234

## Testing notes

- Check `StatusCard` typography and spacing is same as Figma now
- Check new Statuses are shown and working fine on `StatusCard` doc
- Check `Tooltip` is displayed on hover of icon if `title` prop exists

## Author checklist before assigning a reviewer

- [x] Reviewed my own code-diff.
- [x] PR assigned to me or an appropriate delegate.
- [x] Relevant labels added to the PR.
- [x] Appropriate tests have been added.
- [x] Lint and test workflows pass.

### CODE REVIEW

File: src/Status/StatusCard/StatusCard.tsx [hunk 1]
```diff
@@ -4,35 +4,35 @@ import { Box, Card, CardContent, Typography } from "@mui/material";
...
   const {
     label: { text }
   } = statuses[status];
   return (
     <CardContent
       sx={{
-        ":last-child": { pb: 2 }
+        ":last-child": { pb: 2 },
+        px: 2.5,
+        py: "14px"
```
Comment (mattcorner) [inline src/Status/StatusCard/StatusCard.tsx:19]: why did we need to add these?

File: src/Status/StatusCard/StatusCard.types.ts [hunk 2]
```diff
@@ -9,4 +9,8 @@ export type StatusCardProps = {
    * The status message.
    */
   name: string;
+  /**
+   * Tooltip text to display on hover of the icon
+   */
+  title?: string;
```
Comment (mattcorner) [inline src/Status/StatusCard/StatusCard.types.ts:15]: `title` sounds like the wrong prop name. `iconTooltipText` sounds better.

File: src/Status/StatusCard/StatusCard.tsx
Comment (mattcorner) [inline src/Status/StatusCard/StatusCard.tsx:1]: The Figma shows that in both light and dark mode the border color of the card has an alpha channel of 23%. However, in storybook they have an alpha channel of 12%.

**Review by mattcorner — CHANGES_REQUESTED** [review]

File: src/Status/StatusCard/StatusCard.tsx (hunk 1 above)
Comment (m4manjesh) [reply src/Status/StatusCard/StatusCard.tsx:19]: Th Figma says py: 14px, px: 24px. So i adjusted slighty to achieve this.
Landing Page – Figma - Google Chrome 9_9_2024 9_49_43 AM (2)

Comment (mattcorner) [reply src/Status/StatusCard/StatusCard.tsx:19]: I think we're still a bit off then. This is what Chrome shows me in Storybook:

image

File: src/Status/StatusCard/StatusCard.tsx
Comment (m4manjesh) [reply src/Status/StatusCard/StatusCard.tsx:1]: Could this be a mistake? 12% is the default coming from MUI and the one we use across the apps.

Comment (m4manjesh) [reply src/Status/StatusCard/StatusCard.tsx:1]: Talked to @jegasriskantha, he agreed to go with 12% default.

**Review by mattcorner — APPROVED** [review]

**Pull request merged**